osso/core/forms/widgets.py

```diff
@@ -4,9 +4,9 @@
 from django.utils.html import conditional_escape
 from django.forms.widgets import CheckboxInput, CheckboxSelectMultiple
 try:
-    from django.utils.encoding import force_text
+    from django.utils.encoding import force_str
 except ImportError:
-    from django.utils.encoding import force_unicode as force_text
+    from django.utils.encoding import force_text as force_str
```

Comment:
So that should need another level of ImportError to fall back to force_unicode, for backwards compatibility?

Reply:
There is more I did on my fork, I am not keen for being that far backwards compatible.
force_str was an alias to force_text for many years, which was marked deprecated and force_unicode should be almost forgotten history by now. Just joined a project using this lib and don't know what for, maybe something with dojo which I am going to make obsolete

Reply:
> I am not keen for being that far backwards compatible.

Okay. But maybe you're not the only one using it.

In this case keeping it backwards compatible seems easy. If I were to commit this, I could break it for the current users.

> Just joined a project using this lib and don't know what for, maybe something with dojo which I am going to make obsolete.

Would that be a telephony project, for a company starting with a G?

Reply:
Yes, that one